src/components/EventTypeFilters.vue

```diff
@@ -1,12 +1,22 @@
 <template>
-  <div class="event-type-filters">
-    <h5 class="event-filters-title">Type of event</h5>
-    <div v-for="(label, type) in eventTypes">
-      <input type="checkbox"
-        class="no-margin-checkbox"
-        :checked="selectedEventTypes.includes(type)" 
-        @change="updateSelectedEventTypes(type, $event)"
-      /> {{label}}
+  <div>
+    <h5 v-if="showTitle" class="filter-events-title">Type of event</h5>
+    <div 
+      class="filter-events-item" 
+      v-for="(label, type) in eventTypes"
+    >
+      <div class="filter-events-checkbox">
```

Comment:
Hm, couldn't we style `input:checked { ... }` instead?

Isn't the point of the label is to include text, though? Right now the label text is outside of the label, which would be a bad experience for screen readers (or anyone who might want a bigger click target than just the checkbox).

If you'd prefer to stick with what you have, that's fine too and I can try to mess with it in a separate PR later. :)